tagstudio/src/core/library/alchemy/models.py

```diff
@@ -177,9 +177,12 @@ def __init__(
         path: Path,
         folder: Folder,
         fields: list[BaseField],
+        id: int | None = None,
     ) -> None:
         self.path = path
         self.folder = folder
+        assert not self.id
```

Comment:
Why is this necessary? Shouldn't the constructor run before anything can set this?

Reply:
I went and followed the assertion that was present in the Entry constructor, but yes I don't think this has ever been tripped during debugging and can probably be taken out now